darts/tests/dataprocessing/transformers/test_data_transformer.py

```diff
@@ -27,6 +29,9 @@ def test_scaling(self):
         series1_tr2 = transformer2.fit_transform(self.series1)
         series3_tr2 = transformer2.transform(self.series3)
 
+        # should keep columns
+        self.assertEqual(self.col_1, series1_tr1.columns)
```

Comment:
Would this have captured the error from before? I'm just asking because as far as I know, integer columns are the default, so if scaler replaced the original column names with integer column names here we would not notice, right?

Reply:
Indeed you are right, I should set names different from integer first.